src/Tribe/Plugins_API.php

```diff
@@ -2,6 +2,32 @@
 
 class Tribe__Plugins_API {
 	/**
+	 * Transient key used to store premium plugin stats.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $cached_stats_key = '_tribe_plugins_api_stats';
+
+	/**
+	 * Our current (or most recent available) plugin statistics, including
+	 * ratings, download count etc.
+	 *
+	 * @var array
+	 */
+	protected $current_stats = array();
+
+	/**
+	 * Endpoint for discovering current plugin statistics.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $stats_url = 'https://m.tri.be/plugin-stats';
```

Comment:
If I strip out that logic, so we aren't connecting to our own domain, do the revised defaults seem reasonable to you? Some of them are clearly fictitious unless we randomize them, and I'm not too sure that's a great idea either.